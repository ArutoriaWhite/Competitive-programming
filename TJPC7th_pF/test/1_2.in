83556781 2
14588599 0
79161415 0
