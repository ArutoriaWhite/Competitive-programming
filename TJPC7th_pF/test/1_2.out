14588599 18983965
